docs/guides/migration_guide_facilities_to_metros_devices.md

```diff
@@ -8,7 +8,7 @@ description: |-
 
 In April 2021, Equinix Metal rolled out a new location concept - [metros](https://feedback.equinixmetal.com/changelog/new-metros-feature-live). A metro is an Equinix-wide concept for data centers which are grouped together geographically. Data centers within a metro share capacity and networking features. You can read more about metros at https://metal.equinix.com/developers/docs/locations/metros/.
 
-Until the metros introduction, resource deployment location used to be controlled by "facilty" - single data center with location code like "sv15" or "ny5". Metros group the facilities, and e.g. metro "sv" contains facility "sv15". If you specify a metro when creating a resource, it will be deployed to one of the facilities in the metro group. You can then find the deployed facility from the a read-only attribute of the resource.
+Until the metros introduction, resource deployment location used to be controlled by "facilty" - single data center with location code like "sv15" or "ny5". Metros group the facilities, so e.g. metro "sv" contains facility "sv15". If you specify a metro when creating a resource, it will be deployed to one of the facilities in the metro group. You can then find the deployed facility from a read-only attribute of the resource.
```

Comment:
```suggestion
Before the introduction of metros, resources were deployed to a single `facility` location.  When provisioning `metal_device` resources, the facility could be chosen by Equinix Metal with a user-supplied list of `facilities`, or a wildcard `any` facility.  The individual facility locations use a code like "sv15" or "ny5". Metros group facilities. For example, metro "sv" contains the "sv15" facility among others. If you specify a metro when creating a resource, it will be deployed to one of the facilities in the metro group. You can then find the deployed facility using a read only attribute of the resource (e.g. `deployed_facility` for `metal_device` resources).
```